docs/about-the-consoles/fader-wings.md

```diff
@@ -24,20 +24,33 @@ The Tiger Touch Wing connects by USB to the main console. It also needs
 a mains power connection. The Wing has a built in USB hub to provide 2
 further USB sockets for a keyboard etc.
 
+A switch on the rear of the wing sets it to left or right
+position, allowing you to attach two of the wings to a console. The switch position
+is only read on power-up.
+
 The Tiger Touch Wing is designed to fit next to the Tiger Touch console
-but can also be used with the Pearl Expert, the Arena and the Sapphire
-Touch.
+but can also be used with the D9, the Sapphire Touch, the Arena and the
+Pearl Expert.
+
+> If you connect the Tiger Touch wing to a D9-330, the left hand faders on the console are the same handle group as the
```

Comment:
Maybe you should say something like "so the same playbacks would appear in both places.